src/network/compressor.h

```diff
@@ -38,23 +38,24 @@
 namespace Network {
   class Compressor {
   private:
-    static const int BUFFER_SIZE = 2048 * 2048; /* effective limit on terminal size */
+    static const int BUFFER_SIZE =
+        2048 * 2048; /* effective limit on terminal size */
 
-    unsigned char buffer[BUFFER_SIZE];
+    unsigned char buffer[ BUFFER_SIZE ];
 
   public:
-  Compressor() : buffer() {}
+    Compressor() : buffer() {}
     ~Compressor() {}
 
     std::string compress_str( const std::string &input );
     std::string uncompress_str( const std::string &input );
 
     /* unused */
     Compressor( const Compressor & );
-    Compressor & operator=( const Compressor & );
+    Compressor &operator=( const Compressor & );
   };
 
-  Compressor & get_compressor( void );
+  Compressor &get_compressor( void );
```

Comment:
Since we were space-&-space before, I think we have an opportunity to do &-space rather than space-&. WDYT?

Reply:
I am pleased to report that the clang-format file that @keithw linked resolves this.